src/write.c

```diff
@@ -886,6 +886,55 @@ static int avifQualityToQuantizer(int quality, int minQuantizer, int maxQuantize
     return quantizer;
 }
 
+// Adds the items for a single cell or a grid of cells.
+// Outputs the mainItemId, being the only cell if there is exactly one, or the grid item for multiple cells.
```

Comment:
Nit: Change "mainItem" to "primaryItem" and "Id" to "ID" to be consistent with current names. Change all occurrences.

Reply:
> Nit: Change "mainItem" to "primaryItem" 

If `alpha` is true, this is not the primary item that is returned, but the "top-level" one: `the only cell if there is exactly one, or the grid item for multiple cells`. That is why I named it "main" instead of "primary".

> "Id" to "ID" to be consistent with current names. Change all occurrences.

This is against Google style but it is not even applied in this project. Done.